sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -116,6 +116,17 @@ abstract class JdbcDialect extends Serializable {
     s"SELECT * FROM $table WHERE 1=0"
   }
 
+  /**
+    * The SQL query that should be used to truncate a table. Dialects can override this method to
+    * return a query that is suitable for a particular database
+    * @param table The name of the table.
+    * @return The SQL query to use for truncating a table
+    */
+  @Since("2.3.0")
```

Comment:
@danielvdende .
As we see here, this is a new feature. It seems that we had better a new JIRA as a `New Feature` with a title like `Add getTruncateQuery to JdbcDialect`.

Reply:
So create a new "improvement" JIRA and change the ticket number in this PR? (just checking I understand you correctly)

Reply:
Yep. And, we need to fix the following.
```
The PostgresDialect indicates that cascade is true by default for Postgres.
This is not the case. This patch fixes this.
```

Reply:
Done